src/components/PeoplePicker/PeoplePicker.tsx

```diff
@@ -75,7 +76,7 @@ export class PeoplePicker extends React.Component<IPeoplePickerProps, IPeoplePic
   }
 
   public componentDidMount() {
-    this._events.on(window, 'blur', this._onBlurCapture, true);
```

Comment:
one of the things i was just chatting with Thomas about... these global events should really only be done when needed. Just mentioning.